site/static/css/asciinema-player.css

```diff
@@ -2561,3 +2561,6 @@
 .asciinema-theme-monokai .fg-15 {
   font-weight: bold;
 }
+.showcase img {
```

Comment:
can you use this file instead https://github.com/GoogleCloudPlatform/agones/blob/master/site/assets/scss/_variables_project.scss , thank you

Reply:
@KamiShepard just making sure you saw this :+1:  looking forward to getting this in.

Reply:
Sorry was on holiday, just pushed a fix!